definition/data/sheets/CaseEvent.json

```
@@ -66,5 +66,6 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "InitiateClaimPaymentCitizen", "Name": "Claim payment initiated", "Description": "Claim payment initiated", "DisplayOrder": 1,"PostConditionState": "awaitingCitizenPayment", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
   {"CaseTypeID": "MoneyClaimCase", "ID": "ResumeClaimPaymentCitizen", "Name": "Claim payment resumed", "Description": "Claim payment resumed", "DisplayOrder": 1,"PreConditionState(s)": "awaitingCitizenPayment","PostConditionState": "awaitingCitizenPayment", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": 0},
   {"CaseTypeID": "MoneyClaimCase", "ID": "CreateCitizenClaim", "Name": "Claim created by citizen", "Description": "Claim created by citizen", "DisplayOrder": 1,"PreConditionState(s)": "awaitingCitizenPayment","PostConditionState": "create", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": 0,"CallBackURLSubmittedEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/submitted", "RetriesTimeoutURLSubmittedEvent": 0},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetPin", "Name": "Reset Pin", "Description": "Reset and send pin to defendant", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": "5,10,15"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ResetPin", "Name": "Reset Pin", "Description": "Reset and send pin to defendant", "DisplayOrder": 1,"PreConditionState(s)": "*","PostConditionState": "*", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutAboutToSubmitEvent": "5,10,15"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ClaimNotes", "Name":  "Claim  notes", "Description": "Claim notes", "DisplayOrder":  1, "PreConditionState(s)":  "*", "PostConditionState": "*", "SecurityClassification": "Public", "ShowEventNotes": "Yes"}
```

```suggestion
  {"CaseTypeID": "MoneyClaimCase", "ID": "ClaimNotes", "Name": "Claim notes", "Description": "Claim notes", "DisplayOrder":  1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public", "ShowEventNotes": "Yes"}
```
(removing some double-spaces)